src/sentry/api/endpoints/sentry_app_details.py

```diff
@@ -18,3 +18,18 @@ def get(self, request, sentry_app):
             return Response(serialize(sentry_app, request.user))
 
         return Response(status=404)
+
+    @requires_feature('organizations:internal-catchall', any_org=True)
+    def put(self, request, sentry_app):
+        serializer = SentryAppSerializer(data=request.DATA, partial=True)
```

Comment:
Maybe we should call these objects `SomethingValidator` instead – we're really only using them for validation. Would reflect their purpose more. I don't really have a strong opinion, just seeing `Serializer` throws me off a bit. Thoughts/opinions?

Reply:
That's fine with me, but maybe we can pull that out into a different PR cause we would probs want to update the sentry app install serializer as well yeah?

Reply:
Yea, +1.